src/main/plugin/iso19115-3.2018/convert/oai_dc.xsl

```diff
@@ -1,21 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<xsl:stylesheet   xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0" 
-						xmlns:gco="http://standards.iso.org/iso/19115/-3/gco/1.0"
-            xmlns:srv="http://standards.iso.org/iso/19115/-3/srv/2.1"
-            xmlns:mds="http://standards.iso.org/iso/19115/-3/mds/2.0"
-            xmlns:mcc="http://standards.iso.org/iso/19115/-3/mcc/1.0"
-            xmlns:mri="http://standards.iso.org/iso/19115/-3/mri/1.0"
-            xmlns:mrs="http://standards.iso.org/iso/19115/-3/mrs/1.0"
-            xmlns:mrd="http://standards.iso.org/iso/19115/-3/mrd/1.0"
-            xmlns:mco="http://standards.iso.org/iso/19115/-3/mco/1.0"
-            xmlns:msr="http://standards.iso.org/iso/19115/-3/msr/2.0"
-            xmlns:lan="http://standards.iso.org/iso/19115/-3/lan/1.0"
-            xmlns:gcx="http://standards.iso.org/iso/19115/-3/gcx/1.0"
-            xmlns:gex="http://standards.iso.org/iso/19115/-3/gex/1.0"
-            xmlns:dqm="http://standards.iso.org/iso/19157/-2/dqm/1.0"
-            xmlns:cit="http://standards.iso.org/iso/19115/-3/cit/2.0"
-						>
+<xsl:stylesheet version="1.0"
+		xmlns:xsl="http://www.w3.org/1999/XSL/Transform"  
+		xmlns:gco="http://standards.iso.org/iso/19115/-3/gco/1.0"
+            	xmlns:srv="http://standards.iso.org/iso/19115/-3/srv/2.1"
+            	xmlns:mds="http://standards.iso.org/iso/19115/-3/mds/2.0"
+            	xmlns:mcc="http://standards.iso.org/iso/19115/-3/mcc/1.0"
+            	xmlns:mri="http://standards.iso.org/iso/19115/-3/mri/1.0"
+            	xmlns:mrs="http://standards.iso.org/iso/19115/-3/mrs/1.0"
+            	xmlns:mrd="http://standards.iso.org/iso/19115/-3/mrd/1.0"
+            	xmlns:mco="http://standards.iso.org/iso/19115/-3/mco/1.0"
+            	xmlns:msr="http://standards.iso.org/iso/19115/-3/msr/2.0"
+            	xmlns:lan="http://standards.iso.org/iso/19115/-3/lan/1.0"
+            	xmlns:gcx="http://standards.iso.org/iso/19115/-3/gcx/1.0"
+            	xmlns:gex="http://standards.iso.org/iso/19115/-3/gex/1.0"
+           	xmlns:dqm="http://standards.iso.org/iso/19157/-2/dqm/1.0"
+            	xmlns:cit="http://standards.iso.org/iso/19115/-3/cit/2.0"
+		exclude-result-prefixes="xsl gco srv mds mcc mri mrs mrd mco msr lan gcx gex dqm cit">
```

Comment:
Maybe easier to use XSL v2 and
```
exclude-result-prefixes="#all"
```
?

Reply:
Would `#all` keep the `xmlns:oai_dc` and `xmlns:dc` declarations in `<oai_dc:dc>` element?

Reply:
I don't see why it would remove a namespace that is used. 

"When a stylesheet uses a namespace declaration only for the purposes of addressing a source tree, specifying the prefix in the [xsl:]exclude-result-prefixes attribute will avoid superfluous namespace declarations in the serialized result tree. " https://www.w3.org/TR/2009/PER-xslt20-20090421/

Reply:
Pushed  a change to use `"#all"`.

Reply:
Not 100% sure but you need to set XSL version to 2.0 to use `#all`

Reply:
Changed to `2.0`